### PR TITLE
Update gen auth docs

### PR DESCRIPTION
**Pull request by petrus-jvrensburg**
This PR updates the moduledoc for `phx.gen.auth` to mention the `--live` option. 

It also updates the _guide_ for `phx.gen.auth`, and links to @bemesa21 's blog post.

### CODE REVIEW

File: lib/mix/tasks/phx.gen.auth.ex [hunk 1]
```diff
@@ -11,7 +11,21 @@ defmodule Mix.Tasks.Phx.Gen.Auth do
 
   Additional information and security considerations are detailed in the
   [`mix phx.gen.auth` guide](mix_phx_gen_auth.html).
-
+
```
Comment (josevalim) [inline lib/mix/tasks/phx.gen.auth.ex:14]:
```suggestion

```

**Comment by chrismccord** [comment]
Please see the changes I pushed earlier today in rc.3 https://github.com/phoenixframework/phoenix/commit/a5e924e1ea5d558614fdcc3799dd9cb65959f411

Note that LV is the default and we prompt for Y/n. Please feel free to expand the changes I made to incorporate your additions. Thanks!

**Comment by petrus-jvrensburg** [comment]
Okay great. Let me rebase this branch and consolidate the changes.

**Comment by chrismccord** [comment]
❤️❤️❤️🐥🔥

**Pull request merged**